guides/Dataset.md

```diff
@@ -286,7 +286,7 @@ An example of a metadata reference to an instance of EML-formatted structured me
       "dateModified":"2019-06-12T14:44:15Z"
     }</strong>
   }
-<pre>
+</pre>
 
 Alternatively, if the schema.org record is meant to describe the metadata record, one could use the inverse property `schema:about` to indicate the linkage back to the Dataset that it describes.  This would be a more rare situation, as typically the schema.org record would be focused on the Dataset itself.
```

Comment:
The choice of `about` vs `subjectOf` has little to do with the intent of the schema.org record. For example, the following are identical except for the direction of the association:
```
{
    "@context": {
        "@vocab": "https://schema.org/"
    },
    "@graph":[
        {
            "@type":"Dataset",
            "@id": "./",
            "subjectOf":[
                {
                    "@id":"./metadata.xml"
                }
            ]
        },
        {
            "@type":"DataDownload",
            "@id": "./metadata.xml"
        }
    ] 
}
```
```
{
    "@context": {
        "@vocab": "https://schema.org/"
    },
    "@graph":[
        {
            "@type":"Dataset",
            "@id": "./"
        },
        {
            "@type":"DataDownload",
            "@id": "./metadata.xml",
            "about":[
                {
                    "@id":"./"
                }
            ]
        }
    ] 
}
```

In each case, the `id` of the metadata can be found by the SPARQL:
```
PREFIX rdf:  <http://www.w3.org/1999/02/22-rdf-syntax-ns#>
PREFIX SO:   <https://schema.org/>

SELECT DISTINCT ?metadata ?join
WHERE {
    ?join rdf:type SO:Dataset .
    ?metadata SO:about|^SO:subjectOf ?join .
}
```

Worked examples of identical `SO:Datasets` are shown on: [SLAB/so_about_example.ipynb at master · datadavev/SLAB · GitHub](https://github.com/datadavev/SLAB/blob/master/SO/so_about_example.ipynb)

There are cases where the `about` approach becomes preferable. One example is the situation where there are multiple data components and multiple metadata documents in a Dataset. In that case, it may be preferable to assert that particular items provide the metadata for other components that are data, i.e. through the `about` assertion rather than the inverse.

Hence, I think it is incorrect to de-emphasize the use of `about` in favor of `subjectOf`. Both are useful and provide the same information.

Reply:
Ok, thanks Dave.  I debated that myself. So, I think its just a matter of expanding the section to fully show the `about` scenario as well. The diagram I made shows both directions and so should be fine I think.

Reply:
Yep, diagram is good.

Reply:
I don't have a comment on the substance of the advice; but I do think there are several places in this section where terms like "can" need to be replaced by "should" or "must".  I will be harping on this a lot, since it seems to be something the repository community needs.

Reply:
Its a question of interoperabiltiy and semantic clarity.  Is the metadata part of the dataset or is it 'meta' -- something about the dataset, not part of the dataset.  I would never expect a  DataDownload of the dataset to return a metadata record about the dataset, but that's what the 'about' example implies. 
Allowing two ways to encode the same information creates problems for writing client software to interpret the metadata. 
I would expect the encoding for the full metadata link to look something like this:
```
{
      "@type":"Dataset",
      "@id": "./",
      "subjectOf": {
            "@type":"CreativeWork",
		"name":"EML metadata describing dataset component",
		"encodingFormat":["application/xml","http://ecoinformatics.org/eml-2.1.1"],
		"url":"./metadata.xml"
            }
  }
```